extensions/measurement-tracking/src/panels/PanelMeasurementTableTracking/index.js

```diff
@@ -258,7 +258,7 @@ function _getDisplayText(
 }
 
 function _round(value, decimals) {
-  return Number(Math.round(value + 'e' + decimals) + 'e-' + decimals);
```

Comment:
I think the above was because the rounding would _drop_ the trailing `0` if it were 0. Does the new approach preserve it?

Reply:
Scratch that. Your change appears to work as expected 👍